extras/rya.streams/query-manager/src/test/java/org/apache/rya/streams/querymanager/QueryManagerIT.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.rya.streams.querymanager;
+
+import org.junit.Before;
+
+public class QueryManagerIT {
+    @Before
+    public void setup() {
+        //
+    }
+
+    // Test No query change log sources, add a new one
+
+    // Test existing query change log sources, add a new one with same rya instance
```

Comment:
I don't think this test makes sense either since you can't know which rya instances are in each source. This does bring up an interesting problem though. I hadn't thought about seeing the same rya instance in more than a single source.